console-subscriber/src/init.rs

```diff
@@ -0,0 +1,83 @@
+use crate::TasksLayer;
+use std::thread;
+use tokio::runtime;
+use tracing_core::{subscriber::Subscriber, Dispatch};
+use tracing_subscriber::{fmt, layer::Layered, prelude::*, EnvFilter, Layer, Registry};
+
+type ConsoleSubscriberLayer =
+    Layered<TasksLayer, Layered<EnvFilter, Layered<fmt::Layer<Registry>, Registry>>>;
+
+/// Starts the console subscriber server on its own thread
+///
+/// This function represents the easiest way to get started using
+/// tokio-console.
+///
+/// ## Configuration
+///
+/// Tokio console subscriber is configured with sensible defaults for most
+/// use cases. If you need to tune these parameters, several environmental
+/// configuration variables are available:
+///
+/// * `TOKIO_CONSOLE_RETENTION_SECS`: The number of seconds to accumulate
+///   completed tracing data. Default: 60s.
+/// * `TOKIO_CONSOLE_BIND`: a HOST:PORT description, such as
+///   localhost:1234 or similar. Default: 127.0.0.1:6669
```

Comment:
nit, take it or leave it:
```suggestion
///   `localhost:1234` or similar. Default: `127.0.0.1:6669`
```

Reply:
addressed in a1ef76b (rebased in concept on the tables)